plumbum/cli/application.py

```diff
@@ -578,8 +578,9 @@ def helpall(self):
 
         if self._subcommands:
             for name, subcls in sorted(self._subcommands.items()):
-                subapp = (subcls.get())("%s %s" % (self.PROGNAME, name))
-                subapp.parent = self
+                subapps = subcls.get()
+                subapps.parent = self
+                subapp = subapps("%s %s" % (self.PROGNAME, name))
```

Comment:
Feel free to update this to the new format syntax, too, would be cleaner:

```python
sub app = subapps("{0} {1}".format(self.PROGNAME, name))
```

(Either is a big readability improvement on the old line, though 👍 )